From: SJ Park <sj@kernel.org>
Date: Sat, 25 Jul 2026 09:52:58 -0700
Subject: [PATCH] mm/internal: update vma_address_end() comment for removed
 vma_address()

vma_address() was replaced with vma_anon_address() and
vma_filebacked_address() by a patch [1].  The comment for
vma_address_end() still mentions the removed function.  Update.

[1] https://lore.kernel.org/20260720-b4-scalable-cow-virt-pgoff-v2-9-2d549757a76f@kernel.org

Signed-off-by: SJ Park <sj@kernel.org>
---
 mm/internal.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/mm/internal.h b/mm/internal.h
index 38b1165212c94..48b6bd3559aee 100644
--- a/mm/internal.h
+++ b/mm/internal.h
@@ -1075,7 +1075,8 @@ static inline unsigned long vma_anon_address(const struct vm_area_struct *vma,
 
 /*
  * At what user virtual address will none of the range be found in vma?
- * Assumes that vma_address() already returned a good starting address.
+ * Assumes that vma_anon_address() or vma_filebacked_address() already returned
+ * a good starting address.
  */
 static inline unsigned long vma_address_end(struct page_vma_mapped_walk *pvmw)
 {
-- 
2.47.3

